src/compiler/commandLineParser.ts

```diff
@@ -434,6 +434,7 @@ namespace ts {
             type: "boolean",
             affectsEmit: true,
             isTSConfigOnly: true,
+            isTsConfigOnlyButAllowsCommandLineFalsy: true,

```

Comment:
In theory, isn't _every_ `isTSConfigOnly` flag allowed to be "unset" by the command line? Like, I can see no reason why `--paths null` shouldn't work tbh.